### PR TITLE
async fetching of address counters

### PR DESCRIPTION
**Pull request by ayrat555**
# Changelog

- async fetching of address counters

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/controllers/address_contract_controller.ex [hunk 1]
```diff
@@ -20,16 +19,13 @@ defmodule BlockScoutWeb.AddressContractController do
...
-
       render(
         conn,
         "index.html",
         address: address,
         coin_balance_status: CoinBalanceOnDemand.trigger_fetch(address),
         exchange_rate: Market.get_exchange_rate(Explorer.coin()) || Token.null(),
-        transaction_count: transaction_count,
-        validation_count: validation_count
+        counters_path: address_path(conn, :address_counters, %{"id" => to_string(address_hash)})
```
[the remark this answers is not in the document]
Comment (ayrat555) [reply apps/block_scout_web/lib/block_scout_web/controllers/address_contract_controller.ex]: you're right. I changed it

**Comment by ayrat555** [comment]
@vbaranov I added the required flag https://github.com/poanetwork/blockscout/pull/2787/commits/e45d1f8d7b2aa2d2a1bbcc0e92b46094c7df635e

**Comment by vbaranov** [comment]
@ayrat555 please resolve merging conflicts

**Comment by ayrat555** [comment]
@vbaranov done

**Pull request merged**